specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/StaticSites.json

```diff
@@ -2747,6 +2876,14 @@
             "$ref": "#/definitions/StaticSiteUserProvidedFunctionApp"
           },
           "readOnly": true
+        },
+        "allowStagingEnvironments": {
```

Comment:
>allowStagingEnvironments [](start = 9, length = 24)

consider using enums instead of bools as per [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean)